.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
why npx instead of npm?

Reply:
`npm` is only for `scripts` section of the `package.json`